civis/io/_tables.py

```diff
@@ -676,10 +676,8 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
         required when appending to an existing table. The table_columns
         parameter is required if the table does not exist, the table is being
         dropped, or the columns in the source file do not appear in the same
-        order as in the destination table.
-        Example:
-        table_columns=[{"name": "foo", "sql_type": "INT"},
-                       {"name": "bar", "sql_type": "VARCHAR"}]
+        order as in the destination table. Example:
+        ``[{"name": "foo", "sql_type": "INT"}, {"name": "bar", "sql_type": "VARCHAR"}]``
```

Comment:
After this PR in the compiled docs
<img width="496" alt="Screen Shot 2021-12-02 at 9 11 32 PM" src="https://user-images.githubusercontent.com/86482098/144619776-e4488c81-782a-470a-907f-8fc2b4d93386.png">
: